libevmasm/Assembly.cpp

```diff
@@ -1625,11 +1627,11 @@ LinkerObject const& Assembly::assembleEOF() const
 				solAssert(index < m_codeSections.size());
 				solAssert(item.functionSignature().argsNum <= 127);
 				solAssert(item.type() == JumpF || item.functionSignature().retsNum <= 127);
-				solAssert(item.type() == CallF || item.functionSignature().retsNum <= 128);
+				solAssert(item.type() == CallF || item.functionSignature().retsNum <= 127);
```

Comment:
No need to check for instruction type here any more:
```suggestion
				solAssert(item.functionSignature().retsNum <= 127);
```

Reply:
done